src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.html

```diff
@@ -48,15 +52,9 @@
           <mat-cell *matCellDef="let element">In Progress</mat-cell>
         </ng-container>
 
-        <!-- Owner Column -->
-        <ng-container matColumnDef="owner">
-          <mat-header-cell *matHeaderCellDef>Owner</mat-header-cell>
-          <mat-cell *matCellDef="let element">{{ element.owner }}</mat-cell>
-        </ng-container>
-
         <!-- Timestamp Column -->
-        <ng-container matColumnDef="createdTimestamp">
-          <mat-header-cell *matHeaderCellDef>Date Created</mat-header-cell>
+        <ng-container matColumnDef="completedTimestamp">
+          <mat-header-cell *matHeaderCellDef>Completed</mat-header-cell>
```

Comment:
OOC what would you show in this column for scenarios added but not completed?

Reply:
I have to double check but I believe it's just a filler "---" or something to that effect

Reply:
fwiw, you'll be armed with a last updated timestamp along with a status (e.g. SUCCESS or not).  If you want only the completed timestamp, (SUCCESS|FAILURE) you probably have to a bit of logic to the .ts file to display it, and/or you will have to dig it out of the result's JSON.

Reply:
Yeah last_updated may make more sense for a column since we also have the same column for planning areas too. I'll ask Scott/Michele if they have thoughts